nexus/library/structure.py

```diff
@@ -766,6 +766,26 @@ def __init__(self,axes=None,scale=1.,elem=None,pos=None,mag=None,
     #end def __init__
 
 
+    def check_consistent(self,tol=1e-8,exit=True,message=False):
```

Comment:
This is true, but it can help for the UI.  The "message" part is only really relevant to a message reporting system; most of the time someone writing a nexus user script will use s.check_consistent(), or "if s.check_consistent(exit=False): ...".  There are a few user facing functions elsewhere in nexus that allow variable output lists depending on user request.